src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/Mutations/useDeleteCollection.ts

```diff
@@ -0,0 +1,58 @@
+import { ConnectionHandler, graphql } from "react-relay"
+import { RecordSourceSelectorProxy } from "relay-runtime"
+import { useMutation } from "Utils/Hooks/useMutation"
+import {
+  useDeleteCollectionMutation,
+  useDeleteCollectionMutation$data,
+} from "__generated__/useDeleteCollectionMutation.graphql"
+
+const deleteCollectionUpdater = (
+  store: RecordSourceSelectorProxy<useDeleteCollectionMutation$data>,
+  data
+) => {
+  const collectionID = data.deleteCollection?.responseOrError?.collection?.id
+
+  const root = store.getRoot()
+  const me = root.getLinkedRecord("me")
+
+  if (!me || !collectionID) {
+    return
+  }
+
+  const otherSavesConnection = ConnectionHandler.getConnection(
+    me,
+    "CollectorProfileSaves2Route_otherSaves"
+  )
+
+  if (!otherSavesConnection) {
+    return
+  }
+
+  ConnectionHandler.deleteNode(otherSavesConnection, collectionID)
+}
```

Comment:
**suggestion**: feel free to ignore if you don't like these changes. if the mutation failed, the updater will still be executed

```javascript
const deleteCollectionUpdater = (
  store: RecordSourceSelectorProxy<useDeleteCollectionMutation$data>,
  data: useDeleteCollectionMutation$data
) => {
  const { responseOrError } = data.deleteCollection ?? {}
  
  if (responseOrError?.__typename !== "DeleteCollectionSuccess") {
    return
  }
  
  const collectionID = responseOrError.collection?.id
  
  ...
}
```